examples/discovery-search-app/cypress/integration/passages/spec.ts

```diff
@@ -101,5 +101,44 @@ describe('Passage Results', () => {
       cy.get('@combinedResult').contains('View passage in document');
       cy.get('@combinedResult').contains('View table in document');
     });
+
+    describe('and clicking on "CI Document" for a result', () => {
+      beforeEach(() => {
+        cy.get('button[data-testid="search-result-element-preview-button"]')
+          .last()
+          .click();
+      });
+
+      it('verify that CI Document loaded correctly', () => {
```

Comment:
nice work! are there any more specifics around "loaded correctly" that could be split into separate `it` statements? seems like a good split might be "it can filter", "it has navigation tabs" or something similar

Reply:
Very true! Just added some clearer sections.